src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -74,42 +74,37 @@ public void RawSendInCaseOfFailure(IModel channel, string address, byte[] body,
 
         public void Initialize(IModel channel, string mainQueue)
         {
-
             CreateExchange(channel, mainQueue);
             channel.QueueBind(mainQueue, mainQueue, string.Empty);
-
         }
 
-        static string ExchangeName(Type type)
-        {
-            return type.Namespace + ":" + type.Name;
-        }
+        static string ExchangeName(Type type) => type.Namespace + ":" + type.Name;
```

Comment:
>A micro-optimisation here would be to use string.Concat(), since the current use of two + operators forces an extra heap allocation.

This already happens automatically: https://ericlippert.com/2013/06/17/string-concatenation-behind-the-scenes-part-one/

For example, here's the IL for `ExchangeName()`:

```
.method private hidebysig static string ExchangeName(class [mscorlib]System.Type 'type') cil managed
{
    .maxstack 8
    L_0000: ldarg.0 
    L_0001: callvirt instance string [mscorlib]System.Type::get_Namespace()
    L_0006: ldstr ":"
    L_000b: ldarg.0 
    L_000c: callvirt instance string [mscorlib]System.Reflection.MemberInfo::get_Name()
    L_0011: call string [mscorlib]System.String::Concat(string, string, string)
    L_0016: ret 
}
```